creator-node/src/services/stateMachineManager/index.js

```diff
@@ -164,8 +161,12 @@ class StateMachineManager {
           })
         }
       }
+    }
 
-      // Start recurring queues that need an initial job to get started
+    if (clusterUtilsForWorker.isThisWorkerFirst()) {
```

Comment:
can the 2 blocks for `isThisWorkerFirst()` be combined into one?

Reply:
I think they can't be combined because it has to clear state if first, then init and set up events (whether or not it's first), and then start queues if first

Reply:
ah right